inventories/vagrant/group_vars/owncloud.yml

```diff
@@ -0,0 +1,28 @@
+---
+owncloud_version: "10.3.1"
+owncloud_fqdn: vagrant.owncloud.demo
+
+# If you would like to access your ownCloud by IP
+# you can add it to trusted domains.
+owncloud_trusted_domains:
+  - "{{ owncloud_fqdn }}"
+  - "{{ ansible_default_ipv4.address }}"
+  - "172.30.1.2"
```

Comment:
Is this IP not assigned inside the Vagrant VM? If so, its maybe duplicated to `"{{ ansible_default_ipv4.address }}"`

Reply:
it is assigned in Vagrantfile manually (hard-coded).

I noticed that `{{ ansible_default_ipv4.address }}` was returning a completely different IP which is not usable

Reply:
Ok, keep it for now.